packages/babel-compiler/package.js

```diff
@@ -0,0 +1,17 @@
+Package.describe({
+  name: "babel-compiler",
+  summary: "Parser/transpiler for ECMAScript 6+ syntax",
+  // Tracks the npm version below.  Use wrap numbers to increment
+  // without incrementing the npm version.
+  version: '5.6.17'
```

Comment:
Don't forget you can also do eg `5.6.17_1` although there are some weird things about underscored versions like that it's hard to properly make prereleases for them.